File: 2.server/src/main/java/ru/vitaliy/petrov/server/forms/requests/userprofile/DriverLicenseCreationRequest.java
package ru.vitaliy.petrov.server.forms.requests.userprofile;

import lombok.Data;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import java.sql.Date;

@Data
public class DriverLicenseCreationRequest {

    @NotNull
    @NotEmpty
    private final String driverLicenseNumber;

    @NotNull
    @NotEmpty
    private final String driverLicenseCategory;

    @NotNull
    private final Date driverLicenseDateOfIssue;
}

File: client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/user/DriverLicenseCreationRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.user

import java.sql.Date

data class DriverLicenseCreationRequest(
    val driverLicenseNumber: String,
    val driverLicenseCategory: String,
    val driverLicenseDateOfIssue: Date
)
